tf_files/configs/kube-services.sh

```diff
@@ -52,10 +56,36 @@ kubectl apply -f services/indexd/indexd-service.yaml
 kubectl apply -f services/gdcapi/gdcapi-service.yaml
 ./services/revproxy/apply_service
 
+if [ "$KUBE_JENKINS" = "enabled" ]; then
```

Comment:
These files get rendered by Terraform so the variables are converted out... need to escape them or turn them into TF variables?

Reply:
nice catch :)